src/cli/commands/create.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+//! Create subnet cli command handler.
+
+use async_trait::async_trait;
+use clap::Args;
+use fvm_shared::clock::ChainEpoch;
+use std::fmt::Debug;
+
+use crate::cli::CommandLineHandler;
+use crate::config::json_rpc_methods;
+use crate::jsonrpc::{JsonRpcClient, JsonRpcClientImpl};
+use crate::server::{CreateSubnetParams, CreateSubnetResponse};
+
+/// The command to create a new subnet actor.
+pub(crate) struct CreateSubnet;
+
+#[async_trait]
+impl CommandLineHandler for CreateSubnet {
+    type Arguments = CreateSubnetArgs;
+
+    async fn handle(arguments: &Self::Arguments) -> anyhow::Result<()> {
+        log::debug!("launching json rpc server with args: {:?}", arguments);
+
+        let params = CreateSubnetParams {
+            parent: arguments.parent.clone(),
+            name: arguments.name.clone(),
+            min_validator_stake: arguments.min_validator_stake,
+            min_validators: arguments.min_validators,
+            finality_threshold: arguments.finality_threshold,
+            check_period: arguments.check_period,
+        };
+
+        let url = arguments.ipc_agent_url.parse()?;
```

Comment:
Can we fallback to the default one if this isn't set?

Reply:
do we have a default value? There is a config file, but the caller can specify any `ipc_agent_url`, should we just let the user config?

Reply:
We are setting a default path for config and optionally we can pass a config path also as argument.